accelerator/config.c

```diff
@@ -13,6 +13,7 @@
 #define CONFIG_LOGGER "config"
 
 static logger_id_t logger_id;
+static int tmp_argc;
```

Comment:
Add comment starting from "FIXME:" to indicate further improvements without depending on temporary variables.